sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -168,11 +169,11 @@ async def read(
         if populate_quota_info is not None:
             request_options["populateQuotaInfo"] = populate_quota_info
 
-        collection_link = self.container_link
-        self._properties = await self.client_connection.ReadContainer(
-            collection_link, options=request_options, **kwargs
+        coll_link = self.container_link
```

Comment:
This line isn't necessary right? We can just use `self.container_link` directly in the lines below?

Reply:
Yes, but I did this to avoid having to put a pylint ignore comment for having a line of code that is too long.